client/extensions/woocommerce/state/sites/data/locations/selectors.js

```diff
@@ -52,13 +52,12 @@ export const areLocationsErrored = ( state, siteId = getSelectedSiteId( state )
 
 /**
  * Common "getDependants" logic for all the selectors that operate on a site's locations data.
- * @param {Object} state Whole Redux state tree
- * @param {*} args Rest of the arguments
- * @return {Array} List of dependants for the selector
+ * @param {number} numArgs Number of arguments the selector takes, excluding the Redux state tree and the site ID
+ * @return {function} Function, as expected by the "createSelector" library
```

Comment:
I believe this should still be `@return {Array} List of dependants for the selector`

Reply:
Whoops – ignore!

Is it a general rule that the site ID should be last, or just circumstance? It doesn't appear to be entirely universal: [#](https://github.com/Automattic/wp-calypso/blob/44bc8529dd8d9ad87948a8c8291a9fee01126e92/client/extensions/woocommerce/state/ui/shipping/zones/locations/selectors.js#L102), [#](https://github.com/Automattic/wp-calypso/blob/44bc8529dd8d9ad87948a8c8291a9fee01126e92/client/extensions/woocommerce/state/ui/shipping/zones/locations/selectors.js#L184)

Reply:
> Is it a general rule that the site ID should be last, or just circumstance?

I've always tried to follow that pattern (well, clearly not always, judging by those examples you pulled haha) because `siteId` is usually the only optional parameter. So most (or all) selector definitions will end with a `siteId = getSelectedSiteId(state)` so they can be used without worrying about the site, but at the same time they are prepared for multi-site scenarios.